tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -316,6 +316,32 @@ def test_return_timestamps_in_preprocess(self):
                 "chunks": [{"text": " Conquered returned to its place amidst the tents.", "timestamp": (0.0, 3.36)}],
             },
         )
+        res = pipe(sample["audio"]["array"], return_timestamps="word")
+        # fmt: off
+        # Note that the word-level timestamps predicted here are pretty bad.
+        self.assertEqual(
+            res,
+            {
+                "text": " Conquered returned to its place amidst the tents.",
+                "chunks": [
+                    {
+                        'timestamp': (0.0, 3.36),
+                        'text': ' Conquered returned to its place amidst the tents.',
+                        'words': [
+                            {'text': ' Conquered', 'timestamp': (29.78, 29.9)},
+                            {'text': ' returned', 'timestamp': (29.9, 29.9)},
+                            {'text': ' to', 'timestamp': (29.9, 29.9)},
+                            {'text': ' its', 'timestamp': (29.9, 29.9)},
+                            {'text': ' place', 'timestamp': (29.9, 29.9)},
+                            {'text': ' amidst', 'timestamp': (29.9, 29.9)},
+                            {'text': ' the', 'timestamp': (29.9, 29.9)},
+                            {'text': ' tents.', 'timestamp': (29.9, 29.9)}
```

Comment:
Given the audio example in this test and the chosen chunk / stride length, this is what the model outputs. They are perfectly valid predictions, just really bad ones. 

It's dangerous to try to match these to the whole sentence timestamps since they are not guaranteed to be correct either (and often aren't), and they're produced by a totally different method.